introduces "properties" and deprecates "attributes" (#16)
algokittens/arc69: README.md

            "type": "string",
            "description": "A URI pointing to a high resolution version of the asset's media."
        },

>         "attributes": {

Reviewer: Could we keep this here, but move it to the bottom? In the description, we could say it's deprecated and should not be used for new NFTs, but that marketplaces need to support both to be fully compliant

            "type": "object", 

>             "description": "Properties following the [EIP-1155]("https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1155.md#erc-1155-metadata-uri-json-schema") "simple properties" format."

Reviewer: ```suggestion
            "description": "Properties following the [EIP-1155](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1155.md#erc-1155-metadata-uri-json-schema) 'simple properties' format."

```

  "description": "arc69 theme song",
  "external_url": "https://www.youtube.com/watch?v=dQw4w9WgXcQ",
  "mime_type": "video/mp4",
  "properties": {
    "Bass":"Groovy", 
    "Vibes":"Funky", 
    "Overall":"Good stuff"

>     }

Reviewer: ```suggestion
  }
```

Managers MAY make an ASA's ARC69 immutable. To do so, they MUST remove the ASA's manager address with an `acfg` transaction.

##### ARC69 attribute deprecation

> The initial version of ARC69 followed the [Open Sea attributes format]("https://docs.opensea.io/docs/metadata-standards#attributes"). As illustrated below:

Reviewer: ```suggestion
The initial version of ARC69 suggest the [Open Sea `attributes` array format]("https://docs.opensea.io/docs/metadata-standards#attributes"). As illustrated below:

```
